dfa/utils.py

```diff
@@ -38,3 +38,17 @@ def child_map(word_path):
     paths_ = tree.iddfs(max_depth=max_length, randomize=randomize)
 
     return (word for word, path in paths_ if end is None or path[-1] == end)
+
+def check_equivalence(dfa_a_, dfa_b_):
```

Comment:
Maybe rename to `find_equiv_counterexample`

Reply:
also, are the trailing underscores intentional? Would prefer `dfa1` or `dfa_a` or even `left` and `right`